tensilelite/Tensile/bin/TensileCreateLibrary

```diff
@@ -24,21 +23,30 @@
 #
 ################################################################################
 
-# This script only gets called by CMake
-
-try:
-    from Tensile import TensileCreateLibrary
-except ImportError:
-    import os.path
-    import sys
-    parentdir = os.path.normpath(os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", ".."))
-    print(parentdir)
-    sys.path.append(parentdir)
+relative2absolute()
+{
+    local relative_path="$1"
+    local absolute_path
+    cd "$(dirname "$relative_path")" || exit
+    absolute_path=$(pwd)/$(basename "$relative_path")
+    cd - > /dev/null || exit
+    echo $absolute_path
+}
 
-    from Tensile import TensileCreateLibrary
+ROCM_PATH="${ROCM_PATH}"
+if [ -z "$ROCM_PATH" ]; then
+    ROCM_PATH="/opt/rocm"
+fi
+WORKING_DIR=/tmp
+ROCISA_LIB_DIR=$(relative2absolute $WORKING_DIR)/rocisa/lib
+SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
+ROCISA_DIR=$SCRIPT_DIR/../../rocisa
+### Build rocisa
+# Currently the path to rocisa is fixed
+mkdir -p $WORKING_DIR/rocisa
+cmake -DCMAKE_CXX_COMPILER=$ROCM_PATH/bin/amdclang++ -DCMAKE_PREFIX_PATH=$(python3 -m nanobind --cmake_dir) -DCMAKE_BUILD_TYPE=Debug -DPYTHON_VERSION=$(python3 --version 2>&1 | cut -d ' ' -f 2) -S $ROCISA_DIR -B $WORKING_DIR/rocisa
```

Comment:
Do we really want to have cmake be a dependency here and attempt to do a build every time TensileCreateLibrary is called? Shouldn't this be pre-built already and just installed as a python module?

Reply:
No, it won't build every time unless you change the code.